.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- According to the project requirements, your `.gitignore` file should contain `node_modules`. See your node modules as a cache for your project's dependencies. Please try to fix this 🙏 
**OPTIONAL** - I advise that you install linters locally on your PC so as to make it easier to detect linter errors. You can follow [these](https://github.com/microverseinc/linters-config/blob/master/html-css/README.md) instructions

Reply:
Thanks for the feedback, i took your advice and initialize the npm first and it generated a node_modules file which the code editor suggested to be moved inside the .gitignore file.